### PR TITLE
Complete javascript-light-bulb exercise-Dulce Herrera

### PR DESCRIPTION
**Pull request by dulceherrera**
https://user-images.githubusercontent.com/97363006/178603000-5662a422-7516-47b9-a5db-d5d596a7c672.mp4

### CODE REVIEW

File: javascript-light-bulb/main.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+var $lightbulb = document.querySelector('.bulb-on');
+var $darklight = document.querySelector('.light-on');
+var counter = 0;
+
+function turnLightBulb() {
+  counter++;
+  if (counter % 2 === 0) {
```
Comment (codingcodymiller) [inline javascript-light-bulb/main.js:7]: A count of the clicks will work here, but you are keeping track of a count and just determining if that count is even or odd, one of two states. A boolean would be a better way to keep track of whether something is one of two different states. You should make a variable to keep track of if the light is currently on, with a boolean value.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: javascript-light-bulb/main.js (hunk 1 above)
Comment (dulceherrera) [reply javascript-light-bulb/main.js]: Screen Shot 2022-07-15 at 4 54 50 PM

File: javascript-light-bulb/main.js [hunk 2]
```diff
@@ -1,10 +1,8 @@
 var $lightbulb = document.querySelector('.bulb-on');
 var $darklight = document.querySelector('.light-on');
-var counter = 0;
 
-function turnLightBulb() {
-  counter++;
-  if (counter % 2 === 0) {
+function turnLightBulb(value) {
+  if ($lightbulb.matches('div.bulb-off')) {
```
Comment (codingcodymiller) [inline javascript-light-bulb/main.js:5]: This is now making your code dependent on the classes currently applied to DOM elements, which means your JavaScript is reliant on HTML to know what to do. You should follow my original suggestion of using a variable containing a boolean value to determine if the light is on or not.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: javascript-light-bulb/main.js (hunk 1 above)
Comment (dulceherrera) [reply javascript-light-bulb/main.js]: Screen Shot 2022-07-25 at 3 45 36 PM

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**